src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.html

```diff
@@ -0,0 +1,70 @@
+<h1>{{pageTitle}}</h1>
+<p>To add a new attachment, cancel an existing attachment, or change an existing attachment you must provide the appropriate effective or cancellation date.</p>
+<hr/>
+
+<form [formGroup]="formGroup">
+
+  <div class="form-group">
+    <common-radio formControlName="attachmentType"
+                  name="attachmentType"
+                  label="What change are you making for Business Cost Premium to the facility?"
+                  [radioLabels]='radioItems' 
+                  display="table-row-group"
+                  (ngModelChange)='changeAttachmentType($event)'
+                  [errorMessage]="sameMailAddrError">
+    </common-radio>
+  </div>
+
+  <ng-container *ngIf='shouldShowNewSection'>
+    <div class="form-group">
+      <common-radio formControlName="newAttachmentType"
+                    name="newAttachmentType"
+                    label="Is this attachment a Locum or temporary?"
+                    [radioLabels]='[{label:"Yes", value: true},{label: "No", value: false}]' 
```

Comment:
common-radio default labels to yes/no.

          <common-radio name="newAttachmentTyp"' label="Is this attachment a Locum or temporary?"
                        formControlName="newAttachmentType"></common-radio>

Reply:
Done.